element_array_ephys/ephys_precluster.py

```diff
@@ -695,12 +861,26 @@ def make(self, key):
 
 @schema
 class WaveformSet(dj.Imported):
+    """A set of spike waveforms for units out of a given CuratedClustering.
+
+    Attributes: 
+        CuratedClustering (foreign key): CuratedClustering primary key. 
+    """
+
     definition = """
     # A set of spike waveforms for units out of a given CuratedClustering
     -> CuratedClustering
     """
 
     class PeakWaveform(dj.Part):
+        """Mean waveform across spikes for a given unit. 
+
+        Attributes:
+            master (foreign key): WaveformSet primary key. 
```

Comment:
I don't think `master` will make sense to new users in the context of API docs. Please change across schema
```suggestion
            WaveformSet (foreign key): WaveformSet primary key. 
```